### DISCUSSION TITLE
Wrong signature with Email login and Smart account

### DISCUSSION THREAD

**Discussion by wallneradam** [reown-com/appkit#3348] [Q&A]
When you select Email account, the below code works as expected (youll get a 65 byte response), but when you select smart account (which is the default now for new users, you got an extreme long and wrong response for signing message.

This is the code to reproduce:
```javascript
import {WagmiProvider, useSignMessage} from 'wagmi'
import {QueryClient, QueryClientProvider} from '@tanstack/react-query'

import {polygon} from '@reown/appkit/networks';

import {createAppKit} from '@reown/appkit/react'
import {WagmiAdapter} from '@reown/appkit-adapter-wagmi'

import './App.css'


const walletConnectProjectId = "123..."
const walletConnectMetadata = {
    name: 'Test',
    description: 'This is a test',
    url: 'https://app.test.io',
    icons: ['https://aoo.test.io/favicon.ico'],
}


const wagmiAdapter = new WagmiAdapter({
    networks: [polygon],
    projectId: walletConnectProjectId,
});


createAppKit({
    adapters: [wagmiAdapter],
    networks: [polygon],
    projectId: walletConnectProjectId,
    metadata: walletConnectMetadata,
});


const queryClient = new QueryClient();

function Buttons() {
    const {signMessageAsync} = useSignMessage();

    const btnSign = async () => {
        let signResponse = await signMessageAsync({
            message: "0xedb93BCF31ADe6f87123561C346efa88c8f05116/1732722941"
        })
        console.log(signResponse);
        console.log(signResponse.length);
    }

    console.log(wagmiAdapter.wagmiConfig)

    return (<>
        <w3m-button/>
        <br/>
        <button onClick={btnSign}>
            Sing Test
        </button>
    </>)
}


function App() {
    return (
        <WagmiProvider config={wagmiAdapter.wagmiConfig} reconnectOnMount={true}>
            <QueryClientProvider client={queryClient}>
                <Buttons/>
            </QueryClientProvider>
        </WagmiProvider>
    )
}

export default App
```
1st am I doing something wrong? Is it a bug?

Is there any way to disable this smart account? It is very annoying for the users to have 2 different account. If they change that by mistake they think we had stolen there crypto. It would be good if it would have the same address, or they could have access their existing data there automatically...

**Comment by arein** [comment]
> Is there any way to disable this smart account? It is very annoying for the users to have 2 different account. If they change that by mistake they think we had stolen there crypto. It would be good if it would have the same address, or they could have access their existing data there automatically...

@wallneradam can you DM me?

I think there's other improvements we can make. Disabling Smart Account sounds like a nuke option to me - and there's many associated user benefits that we wouldn't want to drop on the floor

**Reply by wallneradam** [reply]
@arein I've DMed you by email, thanks

**Comment by lordrio** [comment]
@wallneradam did you found a way to fix this? I'm stuck with the same issue as well.
